tests/Set/DoctrineORM29Set/Fixture/join_columns.php.inc

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Rector\Doctrine\Tests\Set\DoctrineORM29Set\Fixture;
+
+use Doctrine\ORM\Mapping as ORM;
+
+class Entity
+{
+    /**
+     * @ORM\JoinColumns({
+     *   @ORM\JoinColumn(name="entity_id", referencedColumnName="id"),
+     *   @ORM\JoinColumn(name="entity_type", referencedColumnName="entity_type"),
+     * })
+     **/
+    protected $page;
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Doctrine\Tests\Set\DoctrineORM29Set\Fixture;
+
+use Doctrine\ORM\Mapping as ORM;
+
+class Entity
+{
+    #[\Doctrine\ORM\Mapping\JoinColumns(['(name="entity_id", referencedColumnName="id")', '(name="entity_type", referencedColumnName="entity_type")', '}'])]
```

Comment:
@zairigimad please verify if this is correct, as it transform: 

```php
    /**
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="entity_id", referencedColumnName="id"),
     *   @ORM\JoinColumn(name="entity_type", referencedColumnName="entity_type"),
     * })
     **/
```

to 

```php
    #[\Doctrine\ORM\Mapping\JoinColumns(['(name="entity_id", referencedColumnName="id")', '(name="entity_type", referencedColumnName="entity_type")', '}'])]

```

Reply:
It seems ok for the transformation, but I'm not sure if the JoinColumns is supported by Doctrine, I cannot find any example for the attribute.